api/mesh/options.proto

```diff
@@ -27,6 +27,12 @@ message KumaResourceOptions {
 
   KumaKdsOptions kds = 10;
   KumaWsOptions ws = 7;
+
+  // Whether scope is "Namespace"; Otherwise to "Cluster".
+  bool scope_namespace = 11;
```

Comment:
Is there a relationship between kuma scope and kubernetes scope? For example, do we have a rule that global kuma scope objects are cluster kubernetes scope?

Reply:
Not that I could discern. I tried using the global to identify namespace and got results that differed in autogen from existing code.